addon/components/polaris-breadcrumbs.js

```diff
@@ -16,7 +10,7 @@ export default class PolarisBreadcrumbs extends Component {
    * @default []
    * @public
    */
-  breadcrumbs = [];
```

Comment:
wouldn't removing this default blow up if breadcrumbs is undefined and we assume it's an array?

Reply:
Not in this case since we only use it in `get breadcrumb()` which won't run unless `breadcrumbs` changes (a.k.a. gets passed in as a value). But to your point maybe we should keep it in case things change in the future?

Should we create a getter with `this.args.breadcrumbs || []`, or just do a check inside `get breadcrumb()` since that's the only spot it's used? Any strong opinion here? @vladucu